cogboard-webapp/src/components/widgets/types/LogViewerWidget/LogList/styled.js

```diff
@@ -144,3 +148,29 @@ export const QuarantineSimilarLogsButton = styled(IconButton)`
 QuarantineSimilarLogsButton.defaultProps = {
   size: 'small'
 };
+
+export const StyledCopyButton = styled(IconButton)`
+  position: absolute;
+  &&& {
+    top: 0.1rem;
+    right: -1rem;
```

Comment:
to by trzeba było przesunąć minimalnie mocniej jeszcze, bo jest za blisko tekstu